meta-balena-common/recipes-core/packagegroups/packagegroup-resin.bb

```diff
@@ -17,6 +17,7 @@ RDEPENDS:${PN} += " \
     dosfstools \
     mobynit \
     docker-disk \
+    crun \
     hostapp-update \
```

Comment:
Given that this is added unconditionally, what is the rootfs size increase?

Reply:
@alexgg Looks like about 476K increase, based on the buildhistory diff

```
-IMAGESIZE = 675296
+IMAGESIZE = 675772
```

Additionally, it appears the source files mentioned below aren't included in the image after all. Here's the full output from the buildhistory diff: https://gist.github.com/jakogut/dd575d995afddfde90fc0e29f2e109b2